courses/free-courses-pl.md

```diff
@@ -80,6 +80,7 @@
 
 * [Kurs JavaScript. Programowanie frontendowe (VIDEO)](https://www.youtube.com/playlist?list=PLOYHgt8dIdoxTUYuHS9ZYNlcJq5R3jBsC) - Mirosław Zelent, Damian Stelmach
 * [Programowanie w JavaScript od podstaw w 1 miesiąc](https://www.youtube.com/playlist?list=PLTs20Q-BTEMPRSzhrlAuu7yus1BuOLVrS)
+* [Zostań Programistą JavaScript](https://youtube.com/playlist?list=PLaRAejmsc8gGAs-Ml8aa4eLCkm6ESvdnN&si=aGm9u8lJl4B1_Fh3) - Kacper Szarkiewicz
```

Comment:
```suggestion
* [Kurs Javascript: Moduł 1: Poczatkujacy](https://youtube.com/playlist?list=PLaRAejmsc8gGAs-Ml8aa4eLCkm6ESvdnN&si=aGm9u8lJl4B1_Fh3) - Kacper Szarkiewicz
```

Reply:
Dont invent titles just put the title given in resource

Reply:
also, use www.youtube.com and remove the si parameter